build.gradle

```diff
@@ -1,6 +1,6 @@
 plugins {
     id "java"
-    id "edu.wpi.first.GradleRIO" version "2023.1.1"
+    id "edu.wpi.first.GradleRIO" version "2023.3.2"
     id "com.peterabeles.gversion" version "1.10"
```

Comment:
GradleRIO should be updated in the main branch quickly. Several Pull Requests have updated it to 2023.2.1. The change is appropriate for more than just vision, so it should be a separate request.

https://plugins.gradle.org/plugin/edu.wpi.first.GradleRIO shows that your 2023.3.2 is the latest.

Make a separate Pull Request that is just this file. That PR should be approved quickly.

Reply:
I've updated main to the latest.